packages/slice-machine/lib/builders/CustomTypeBuilder/TabModal/update.tsx

```diff
@@ -20,7 +20,7 @@ const InputBox = ({
   placeholder: string;
   error?: string;
   [x: string]: any;
-}) => (
+}): React.ReactElement => (
```

Comment:
It's better to add React.FunctionComponent to the InputBox to type the returned object and props at the same time :)